src/sass/components/_contacts.sass

```diff
@@ -1,27 +1,26 @@
 .contacts-wrapper 
-	width: 100%
-	background-color: $black
   background: linear-gradient(0deg, rgba(0, 0, 0, 0.75), rgba(0, 0, 0, 0.75)), url(../../img/header-bg-light.jpg)
+  background-size: cover, cover
   
  
   
 .contacts
-	font-weight: 700
+	font-weight: bold
```

Comment:
Better remove this line.
`font-weight: 700` applied to `html`, and this block inherits this property.

Reply:
I thought 700 is bolder than bold. Removed that line anyway

Reply:
`font-weight: 700` === `font-weight: bold`